judgments/views/detail.py

```diff
@@ -79,24 +84,22 @@ def detail(request, document_uri):
     context = {}
 
     press_summary_suffix = "/press-summary/1"
-    if document_uri.endswith(press_summary_suffix):
-        context["document_type"] = "press_summary"
-        context["linked_document_uri"] = document_uri.removesuffix(press_summary_suffix)
-        context["judgment_ncn"] = ""
+    context["document_noun"] = document.document_noun
+    if document.best_human_identifier is None:
+        raise NoNeutralCitationError(document.uri)
+    context["judgment_ncn"] = document.best_human_identifier
+    if document.document_noun == "press summary":
+        linked_doc_url = document_uri.removesuffix(press_summary_suffix)
     else:
-        context["document_type"] = "judgment"
-        context["linked_document_uri"] = document_uri + press_summary_suffix
-        context["judgment_ncn"] = document.neutral_citation
+        linked_doc_url = document_uri + press_summary_suffix
 
-    linked_document = None
     try:
-        linked_document = get_published_document_by_uri(context["linked_document_uri"])
+        linked_document = get_published_document_by_uri(linked_doc_url)
+        context["linked_document_uri"] = linked_document.uri
     except Http404:
+        linked_document = None
```

Comment:
so now we don't need `linked_document` as a variable in the except case as it is now onlyu usef used for `linked_document_uri` so we can remove this line 100 `linked_document = None`